setup.py

```diff
@@ -1,8 +1,24 @@
 from setuptools import setup
 
+
+with open('README.rst') as readme:
+    long_description = readme.read()
+
 setup(
     name='edx-repo-tools',
```

Comment:
@jmbowman is this name fine for PyPi?

Reply:
Sure, shouldn't conflict with anything else and I doubt if we'd ever want to use that name for a different package.